AppRun

```diff
@@ -1,5 +1,14 @@
-#!/bin/sh
+#!/bin/bash
 
-export PATH="$(dirname $0)/usr/bin:$PATH"
+if [[ -z "$APPDIR" ]]; then
+    export APPDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
+fi
+export PATH="$APPDIR/usr/bin:$PATH"
+export LD_LIBRARY_PATH="$APPDIR/usr/lib/:$LD_LIBRARY_PATH"
+export XDG_DATA_DIRS="$APPDIR/usr/share/:$XDG_DATA_DIRS"
 
-curv $@
\ No newline at end of file
+if [[ -z "$@" ]]; then
+    curv -le new.curv
```

Comment:
I don't like this code because `curv` with no arguments should invoke the interactive command line REPL. I think you are solving the problem of how should curv behave when launched as a GUI from an App menu in a desktop environment, which is a use case I haven't worried about in the past. I would prefer to solve this problem by adding a `-g` command line option to run in "GUI" mode.

Reply:
You're right that the AppImage should behave exactly the same as the curv binary in the command line and therefore should not implement some kind of default argument. I kinda had thought of this before but I don't know why I still ended up doing the contrary in the AppRun.
That solution using the desktop file is probably the best one.

Reply:
Btw, curv -le does require filename. It complains "missing filename argument" if I just do "curv -le".

Reply:
What I meant was, modify `curv/curv.cc` so that `curv -le` does not require a filename argument. In other words, move your new logic out of `AppRun` and into `curv.cc`. Doesn't need to be done for this PR, do it later.